<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="insMax"/>
     </item>
     <item>
      <widget class="QPushButton" name="insBMax">
       <property name="text">
        <string>Добавить элемент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exMax"/>
     </item>
     <item>
      <widget class="QPushButton" name="exBMax">
       <property name="text">
        <string>Извлечь максимум</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="getMax"/>
     </item>
     <item>
      <widget class="QPushButton" name="getBMax">
       <property name="text">
        <string>Получить максимум</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="prMax"/>
     </item>
     <item>
      <widget class="QPushButton" name="prBMax">
       <property name="text">
        <string>Вывести кучу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Куча на минимум</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Куча на максимум</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>331</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="insMin"/>
     </item>
     <item>
      <widget class="QPushButton" name="insBMin">
       <property name="text">
        <string>Добавить элемент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exMin"/>
     </item>
     <item>
      <widget class="QPushButton" name="exBMin">
       <property name="text">
        <string>Извлечь минимум</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="getMin"/>
     </item>
     <item>
      <widget class="QPushButton" name="getBMin">
       <property name="text">
        <string>Получить минимум</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="prMin"/>
     </item>
     <item>
      <widget class="QPushButton" name="prBMin">
       <property name="text">
        <string>Вывести кучу</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
